### PR TITLE
Pyrometer

### PR DESCRIPTION
**Pull request by nicolasochem**
This is a chart I'm using to deploy pyrometer in teztnets: https://pyrometer.teztnets.xyz

So, it's known working.

### CODE REVIEW

File: charts/pyrometer/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    matchLabels:
+      app: pyrometer
+  template:
+    metadata:
+      labels:
+        app: pyrometer
+      annotations:
+        # ensure that the pod bounces each time configmap changes
+        # https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```
Comment (harryttd) [inline charts/pyrometer/templates/deployment.yaml:17]: very cool

File: charts/pyrometer/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+data:
+  config.json: |
+{{ .Values.config | mustToPrettyJson | indent 4 }}
```
Comment (harryttd) [inline charts/pyrometer/templates/configmap.yaml:4]: can we default to `{}` so if `config` value is empty/null it won't break pyrometer. A null value breaks it

**Review by harryttd — CHANGES_REQUESTED** [review]

File: charts/pyrometer/templates/configmap.yaml (hunk 2 above)
Comment (nicolasochem) [reply charts/pyrometer/templates/configmap.yaml:4]: why would you launch pyrometer without config thou?

Comment (harryttd) [reply charts/pyrometer/templates/configmap.yaml:4]: ah right. good point. There are no defaults it would fall back on.

**Review by harryttd — APPROVED** [review]

**Pull request merged**